.dockerignore

```diff
@@ -0,0 +1,6 @@
+node_modules
+.git
+.gitignore
+*.md
+dist
+dev-dist
```

Comment:
some other ideas for things we can remove

* `*.spec.ts` files i don't think we need to copy any of the tests over
* maybe just a good idea to add permutations of `.env` files just to prevent anything crazy from happening

Reply:
good point! will add